tests/conformance/conformance_test.go

```diff
@@ -1925,6 +1925,35 @@ var internalTestCases = []testCase{
 		shouldFailAt: 2,
 	},
 
+	{
+		name: "copy-multiple-missing-file with glob",
+		dockerfileContents: strings.Join([]string{
+			"FROM scratch",
+			"COPY file-z.txt subdir-*",
+		}, "\n"),
+		contextDir:   "dockerignore/populated",
+		shouldFailAt: 2,
+	},
+
+	{
+		name: "copy-multiple-missing-file with no match on glob",
+		dockerfileContents: strings.Join([]string{
+			"FROM scratch",
+			"COPY missing*",
```

Comment:
There should be a destination on this line.  Are you sure it fails to build because of a globbing error?